src/assets/source/scripts/modules/serlo_injections.js

```diff
@@ -68,6 +75,40 @@ define(['jquery', 'common', 'translator', 'content'], function ($, Common, t) {
                 }
             }
 
+            function initGeogebraTube() {
+                //console.log(href.substr(0, 34));
+                if (href.substr(0, 34) !== "http://tube.geogebra.org/student/m") {
+                    return false;
+                }
+                var gtAppletID = 'gtApplet' + gtAppletsCount, applet, transform,
+                    $clone = $geogebraTubeTemplate.clone();
+
+                //if (gtAppletsCount === 0)
+                //    require([geogebraTubeScriptSource]);
+                gtAppletsCount += 1;
+
+                $clone.attr('id', gtAppletID);
+
+                $that.html($clone);
+
+                applet = new GGBApplet({material_id: href.substr(34)}, true);
+
+                applet.inject(gtAppletID, 'preferHTML5');
+
+                transform = function () {
+                    //$clone.find("div:first").css("transform", "none");
+                    $clone.parent().width($clone.find("div:first").width() * $clone.find("div:first > article").attr("data-param-scale"));
+                    $clone.parent().height($clone.find("div:first").height() * $clone.find("div:first > article").attr("data-param-scale"));
+                    //$clone.parent().css("overflow", "hidden");
+                    //console.log($clone.find("div:first > article").attr("data-param-scale"));
+                };
+                setTimeout(transform, 20000);
```

Comment:
sometimes it took pretty long for the GeoGebraTube Applet to load, so this is a precaution.
Maybe 20000 is longer than needed, but it shouldn't create any problems if the first timeout already worked.